system/core/pages.php

```diff
@@ -534,3 +567,60 @@ function image_resize($src, $dst, $width, $height) {
 		}
 		return true;
 	}
+
+	function getButtonInfo($_args) {
```

Comment:
I'm confused by this name, AFAIK it retrieves generic help/tooltip content, not just limited to buttons?